gc/stats/AllocationStats.hpp

```diff
@@ -59,14 +60,22 @@ class MM_AllocationStats : public MM_Base
 
 #if defined(OMR_GC_THREAD_LOCAL_HEAP)
 	uintptr_t tlhBytesAllocated() { return _tlhAllocatedFresh - _tlhDiscardedBytes; }
+	uintptr_t tlhBytesAllocatedUsed() { return _tlhAllocatedUsed; }
```

Comment:
Put some comment here. For example (if you think this is true):

For tlhBytesAllocated, it's allocated bytes since last TLH refresh.
For tlhBytesAllocatedUsed, it's allocated bytes since last OOL allocation from this TLH (which could be forced before TLH refresh multiple times due to allocation sampling).

Reply:
bytesAllocated(true) try to return the size for all of allocated bytes excludes any bytes in current TLH.
tlhBytesAllocated() also includes the current TLH size, since tlhBytesAllocated() has been used for multiple places, I don't want to change the behavior except for traceAllocationBytes.